src/adapters/smartContractQueriesAdapter.ts

```diff
@@ -0,0 +1,44 @@
+import { Address } from "../address";
+import { IAddress } from "../interface";
+import { IContractQueryResponse } from "../interfaceOfNetwork";
+import { SmartContractQuery, SmartContractQueryResponse } from "../smartContractQuery";
+
+interface INetworkProvider {
+    queryContract(query: ILegacyQuery): Promise<IContractQueryResponse>;
+}
+
```

Comment:
Why are the interfaces in this file, it doesn't look like they are in the right place. The names don't have any connection